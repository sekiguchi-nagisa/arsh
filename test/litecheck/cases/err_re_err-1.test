
# RUN:   /bin/sh $self
echo hello success!!
echo 1>&2 hoge
echo AAA ZZZ BBB
# CHECK_RE: hello success!!
# CHECK_RE: .+ B+
# CHECKERR_RE: ^.$