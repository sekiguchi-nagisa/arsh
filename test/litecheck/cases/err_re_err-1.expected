CHECKERR_RE on line 7,
    Expect: ^.$
    Actual: hoge
stderr is mismatched
